wrapper/src/main/java/com/splunk/support/lambda/ExportersInitializer.java

```diff
@@ -51,21 +58,18 @@ static SdkTracerProvider configureExporters(List<String> exporters, Properties c
         spanProcessors.add(exporter);
       }
     }
-    TraceConfig traceConfig = TraceConfig.builder().setSampler(Sampler.alwaysOn()).build();
 
-    SdkTracerProvider result =
-        SdkTracerProvider.builder()
-            .addSpanProcessor(SpanProcessor.composite(spanProcessors))
-            .setTraceConfig(traceConfig)
-            .build();
-    return result;
+    return SdkTracerProvider.builder()
+        .addSpanProcessor(SpanProcessor.composite(spanProcessors))
+        .setSampler(Sampler.alwaysOn())
```

Comment:
do we also need to set the Resource, since Resource.getDefault() no longer includes all the SPI-based resources?

Reply:
Switched to autoconf